templates/containers/what-are-containers.html

```diff
@@ -14,282 +14,255 @@
-                hi_def=True,
-                loading="lazy") | safe
-        }}
+    </div>
+    <div class="u-fixed-width">
+      <div class="p-media-container p-strip is-shallow u-darker-background col-3 u-align--center u-vertically-center">
+        <img width="834"
+             src="https://assets.ubuntu.com/v1/a09a1433-diagram%202.png"
+             alt=""
+             loading="lazy" />
+      </div>
+      <div class="p-strip is-shallow row u-no-padding--bottom">
+        <div class="col-6 col-start-large-7">
+          <p>
+            Docker is the most popular process container environment and allows for very small, immutable containers, containing only the application binaries and a minimal subset of libraries necessary for an application to run as a single process, requiring limited system resources.
+          </p>
+          <p>
+            LXD functions similarly to a traditional hypervisor and machine containers resemble virtual machines. Machine containers, also referred to as ‘system containers’ are larger compared to process containers, as well as stateful and mutable. They have an allocated filesystem, often containing a cut-down version of an operating system, which might include a shell and a limited number of daemons.
+          </p>
+          <p>
+            Docker is mostly used by developers making Platform-as-a-Service application instances more portable. LXD usage, on the other hand, is often driven by DevOps making Infrastructure-as-a-Service OS instances much faster.
+          </p>
+          <p>
+            Docker can run alongside LXD with both instances working together. Moreover, Docker can run inside of LXD with zero performance impact, allowing you to safely migrate your Docker containers between machines for easy scale-up and scale-down
+          </p>
+          <hr class="p-rule" />
+          <p>
+            <a href="lxd">Learn more about LXD&nbsp;&rsaquo;</a>
+          </p>
+        </div>
       </div>
     </div>
   </section>
 
-  <section class="p-strip--suru">
-    <div class="row">
-      <h2>Further reading</h2>
-    </div>
-    <div class="row p-divider is-dark">
-      <div class="col-4 p-divider__block">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            {{ image(url="https://assets.ubuntu.com/v1/271921af-Datasheet+-+white.svg",
-                        alt="",
-                        width="32",
-                        height="28",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},) | safe
-            }}
-            <h3 class="p-heading-icon__title p-heading--4 ">Data sheet</h3>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col-3 col-medium-6">
+        <h2>Further reading</h2>
+      </div>
+      <div class="col-9 col-medium-5 col-start-medium-2">
```

Comment:
```suggestion
    <div class="row--25-75">
      <hr class="p-rule" />
      <div class="col">
        <h2>Further reading</h2>
      </div>
      <div class="col">
```
Sorry, didn't catch this yesterday, but `row--50-50` is for when you want two col-6s by default, looks like that you want here is a 25/75 split. Using [this pattern](https://vanillaframework.io/docs/patterns/grid#common-patterns) would also account for smaller screens without having to define med-cols 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/5fe9ff1d-7c9a-40f5-8fce-937e20c8ac47)

Reply:
This would not work for the nested grid inside the 25-75 split

Reply:
Yup, I was talking specifically about the lines I edited not the nested rows :)

This works fine too though!